pkg/account/downloader/internal/http/client.go

```diff
@@ -41,6 +41,29 @@ func (c *Client) GetUsers(ctx context.Context, accountUUID string) ([]accountman
 	return r.Items, nil
 }
 
+func (c *Client) GetServiceUsers(ctx context.Context, accountUUID string) ([]accountmanagement.ExternalServiceUserDto, error) {
+	serviceUsers := []accountmanagement.ExternalServiceUserDto{}
```

Comment:
nit: I think the convention on an empty slice would be declaring it like `var serviceUsers []accountmanagement.ExternalServiceUserDto`

Reply:
Here I guess it depends what we want: In the case that there are no service users, do we want an empty slice or nil?  I prefer an empty slice.

Reply:
Would a named param be then an option? Just thinking out loud, maybe stupid idea :).